rules-java-archives/addon/pom.xml

```diff
@@ -15,12 +15,31 @@
 
         <!-- Local Dependencies -->
         <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>6.0</version>
+            <scope>compile</scope>
+            <optional>true</optional>
```

Comment:
so this still stays without definition in windup-bom, this is not external dependency in the reality, right ?